protocol/SignalingProtocol.proto

```diff
@@ -111,6 +113,7 @@ message SdkIndexFrame {
     optional bool at_capacity = 1;
     repeated SdkStreamDescriptor sources = 2;
     repeated uint32 paused_at_source_ids = 3;
+    optional uint32 num_participants = 4;
```

Comment:
What is this used for?

Reply:
This field was generated from the most current service protocol spec. It is appearing now since it had not been synced to the JS SDK protocol before now.